tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -203,6 +203,15 @@ public static function convertColumnProvider()
                 ['type' => 'BYTEA'],
                 ['type' => 'binary', 'length' => null]
             ],
+            // Binary
+            [
+                ['type' => 'BIT VARYING(3)'],
+                ['type' => 'binary', 'length' => 3]
+            ],
+            [
+                ['type' => 'BIT(1)'],
+                ['type' => 'binary', 'length' => 1]
```

Comment:
Bit fields in postgres are odd as they are represented with strings. I am not sure mapping them to binary will give folks the results they are expecting.

Reply:
Sounds reasonable